modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ManagementToolbar.tsx

```diff
@@ -162,8 +164,15 @@ export default function ManagementToolbar({
 			{visibleModal && (
 				<ModalEditExternalReferenceCode
 					externalReferenceCode={externalReferenceCode}
+					getEntity={() =>
+						API.getObjectDefinitionById(objectDefinitionId)
```

Comment:
Hey @naraandrade, shouldn't we use the endpoint to get the definition by ERC here too?

Reply:
We can't do it, @danvb112. It would work just for the first time we open the ModalEditExternalReferenceCode, but from the second time, it will not save correctly. I can take more time later to try to fix it, but for now, we'll have to keep it this way.

Reply:
Ok @naraandrade, Thanks for explain!